### PR TITLE
Feature/jdf/style input

### PR DESCRIPTION
**Pull request by jfrankl**
### Things fixed/changed

Added icon font for transit icons and added font copy process to gulpfile.

Added transit icons and rearranged text to sentence-based style:

image

Fixed issue where typeahead text was appearing out of place:

image

Fixed issue where input fields on homepage were really big:

image

### CODE REVIEW

**Comment by flibbertigibbet** [comment]
We have Font Awesome and Material Design fonts loaded in the project already; the Material Design maps icons are being used in the directions list.  Shall we use the Material Design icons for the selectors added here to the options, or swap out the directions icons for new ones?  We should probably stick with one set, for consistency.

File: python/cac_tripplanner/templates/home.html [hunk 1]
```diff
@@ -22,14 +22,24 @@ <h1 class="text-center">Eco-friendly exploration of Greater Philadelphia on <em>
...
             <span class="text">using</span>
-            <select id="exploreMode" class="selectpicker">
-                <option value="TRANSIT,WALK" selected>Transit</option>
-                <option value="BICYCLE">Bike</option>
-                <option value="BICYCLE,TRANSIT">Bike + Transit</option>
-                <option value="WALK">Walk Only</option>
-                <option value="BUSISH,WALK">Transit (Prefer Bus)</option>
-                <option value="TRAINISH,WALK">Transit (Prefer Trains)</option>
-            </select>
+            <div id="directionsMode" class="btn-toolbar transit-types-container">
```
Comment (flibbertigibbet) [inline python/cac_tripplanner/templates/home.html]: I believe we will need to keep the `selectpicker` class set on all inputs for automatic re-querying to continue to work.

**Comment by jfrankl** [comment]
@flibbertigibbet Swapped out the new icons for the MD icons (good call!) and added ```.selectpicker``` to form fields. Please let me know if anything else needs to be done before merging.

**Comment by flibbertigibbet** [comment]
Okay, great. LGTM.  I'll work on wiring up the new buttons.

**Pull request merged**